zipkin/zipkin/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if [ -n "$STORAGE_TYPE" ]; then
```

Comment:
This doesn't work, but it should.. maybe there is a Dockerfile problem or a CWD one?

```suggestion
if [ -f ".${STORAGE_TYPE}_profile" ]; then
```